src/System.Net.WebClient/tests/WebClientTest.cs

```diff
@@ -54,23 +53,29 @@ public static void DownloadData_InvalidArguments_ThrowExceptions()
             AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataAsync((Uri)null); });
             AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataAsync((Uri)null, null); });
 
-            AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataTaskAsync((string)null); });
+            if (!PlatformDetection.IsFullFramework) // netfx has bug and returns wrong argument name in exception.
+            {
+                AssertExtensions.Throws<ArgumentNullException>("address", () => { wc.DownloadDataTaskAsync((string)null); });
```

Comment:
ok.  I will change this to use the "path" argument when running on NETFX. This way we document the exact expected behavior on NETFX.